app/build.gradle

```diff
@@ -4,6 +4,9 @@
 
 import org.mozilla.gradle.tasks.ValidateAndroidAppReleaseConfiguration
 
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.26"
```

Comment:
Why is this needed? Is this used by the glean script?

Reply:
Yep.  Related docs, FYI: https://mozilla.github.io/glean/book/user/adding-glean-to-your-project.html#adding-new-metrics